### DISCUSSION TITLE
Certificates

### DISCUSSION THREAD

**Discussion by omichel** [cyberbotics/wrestling#94] [Announcements]
We are sending today the certificates to the winners.
The top 3 winner will receive an official ICRA certificate by post.
The other 29 participants selected for the finals will receive an official Cyberbotics digital certificate (PDF).
![IMG_20230605_103743](https://github.com/cyberbotics/wrestling/assets/1264964/38d75adb-edb1-47e4-a0b1-ec10868de843)
![IMG_20230605_103818](https://github.com/cyberbotics/wrestling/assets/1264964/d89b1430-f1b5-400c-9d79-9c9ea16f63cd)
![IMG_20230605_103850](https://github.com/cyberbotics/wrestling/assets/1264964/214eb0ba-1dec-49be-97e0-b65d3a93f6d3)

**Comment by amirhnourian** [comment]
@omichel Great experience and competition! 
Can we double-check the certificate name and family name? 
I changed my JSON file to Amirhossein Nourian
I appreciate your efforts and follow-up for this competition

**Comment by AndrejOrsula** [comment]
Congratulations to all the winners!

**Comment by omichel** [comment]
I sent all the certificates to those who were qualified for the ICRA finals. The certificates are attached to a GitHub issue that I opened on your participant repositories.

**Reply by Henry-Lee24** [reply]
I don't seem to have received my certificate. My team's name is LHL-bot

**Reply by omichel** [reply]
Sorry, I missed this one. It should be [here](https://github.com/Henry-Lee24/LHL-bot/issues/3) now.

**Comment by amirhnourian** [comment]
I am just wondering is there any certification for the other 32 teams for the IROS competition too?

**Reply by omichel** [reply]
Unfortunately, since we were unable to attend IROS2023 in person, there won't be any official certificate this time.

**Reply by MJavadZallaghi** [reply]
Any certificate wont be provided for IROS competition? For example, digital ones by Cyberbotics too?
I see and understand the reason you have mentioned.

**Reply by omichel** [reply]
No certificates for this time. You should refer to the result page to show your participation.
